diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/AvailableRulesDocTest.kt

```diff
@@ -0,0 +1,63 @@
+package org.cqfn.diktat.ruleset.utils
+
+import org.cqfn.diktat.ruleset.constants.Warnings
+import org.junit.jupiter.api.Test
+import java.io.File
+
+/**
+ * Special test to check that developer has not forgotten to write documentation for each warning.
+ * Documentation should be added to available-rules.md
+ */
+class AvailableRulesDocTest {
+    companion object {
+        const val availableRulesFile = "../info/available-rules.md"
+        const val tableDelimiter = "-----"
+        const val ruleNameHeader = "Rule name"
```

Comment:
Maybe, it's also worth checking that rule description is not empty?

Reply:
will add it later